torchrl/envs/vec_env.py

```diff
@@ -514,6 +514,10 @@ def close(self) -> None:
     def _shutdown_workers(self) -> None:
         raise NotImplementedError
 
+    def _set_seed(self, seed: Optional[int]):
+        # This method is not used in batched envs
+        return
```

Comment:
Maybe `pass` instead of `return`? Seems semantically more appropriate to me

Reply:
Yep will fix